cellular_automata_sgs.F90

```diff
@@ -180,13 +199,39 @@ subroutine cellular_automata_sgs(kstep,dtf,restart,first_time_step,sst,lsmsk,lak
       ssti(i,j)          = sst(blk,ix)
       lsmski(i,j)        = lsmsk(blk,ix)
       lakei(i,j)         = lake(blk,ix)
+      dxi(i,j)           = dx(blk,ix)
+      do k = 1,levs
+         heighti(i,j,k) = height(blk,ix,k)/grav
```

Comment:
I suggest getting rid of the division here.  replay /grav with *inv_grav
where inv_grav=1.0/9.81 when it is defined.